index.html

```diff
@@ -644,6 +678,106 @@ <h3>
             </ol>
           </li>
         </ol>
+        <p>
+          When the user agent modifies the list of |active touch points| for a
+          touch surface by adding touch points, removing touch points, or
+          updating the values of existing touch points, <dfn>update
+          touchEvents</dfn> by running the following steps:
+        </p>
+        <ol class="algorithm">
+          <li>Let |surfaceId:unsigned long| be 0.
+          </li>
+          <li>Remove any existing events from
```

Comment:
This seems to assume the gamepad is a live object? I don't think this correct. Each gamepad object is just a snapshot in time.